CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.46.1](https://github.com/rudderlabs/rudder-transformer/compare/v1.46.0...v1.46.1) (2023-10-19)
```

Comment:
@Gauravudia the commit should have been `fix: To make compatible with any server version`
you can update change log here

Reply:
Added